packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -22,6 +22,20 @@
           <context-actions v-if="showContextActions" :items="[currentFolder]" />
         </template>
       </oc-breadcrumb>
+      <!-- needs mobile adjustment -->
+      <div v-if="isLocationSharesActive" class="oc-my-m">
+        <router-link :to="'/files/shares/with-me'" class="oc-px-s">
```

Comment:
You could loop over the sharing related nav items here.

Would prefer to have a sub component for the shares tabs.

Reply:
Yeah that was just a quick "make it work" draft ;) I've created a dedicated component which has a different mobile version, we'll have to discuss a11y topics regarding this one together!